api/series.go

```diff
@@ -94,18 +65,18 @@ func (h *Handler) ListPoints(c *gin.Context) {
 	exists, err := h.PS.Exists(id)
 	if err != nil {
 		logger.Error("Failed to check if series with id "+id+" exists", err)
-		c.JSON(http.StatusInternalServerError, types.APIError{Msg: "Error fetching points, see logs for more info"})
+		c.JSON(http.StatusInternalServerError, types.NewErrorRes("Error fetching points, see logs for more info"))
 		return
 	}
 	if !exists {
-		c.JSON(http.StatusNotFound, types.APIError{Msg: "Series with id " + id + " could not be found"})
+		c.JSON(http.StatusNotFound, types.NewErrorRes("Series with id "+id+" could not be found"))
```

Comment:
'with the ID'

Reply:
Yeah, I missed those 3, fixed and pushed, thanks